pointers/pointers.c

```diff
@@ -6,9 +6,33 @@
     address it's referring to) or the value at the address it's 
     pointing at.
 */
+
+// #include <stdio.h>
+
+// int main(void)
+// {
+// 	int x = 0;   // Set x to 0
+
+// 	int *p = &x; // Set p to point to x, set to address of x
+
+// 	*p = 3490;   // Set the thing that p points to to 3490
+
+// 	printf("x is %d\n", x);  // 3490
+// 	printf("x is %d\n", *p); // 3490 
+// printf("p is %d\n", &p);
+// printf("x is %d\n", &x);
+
+// 	return 0;
+// }
+
+
 void swap(int* a, int* b)
 {
+    *a = &b;
+    *b = &a;
 
+    printf("a is %d\n", *a);
+    printf("b is %d\n", *b);
```

Comment:
IIRC, `swap` should run so that the value stored in `b`'s memory address will be now at `a`'s memory address, and vice versa. If I'm parsing this correctly, you're saying:
```
*a = &b; // let the _value_ of a be the _memory address_ of b
*b = &a; // let the _value_ of b be the _memory address_ of a
```
Is that your intention?